src/index.js

```diff
@@ -10,13 +10,13 @@ const moment = require('moment');
 
 
 let selectionBox = document.querySelector('.selection-box');
-let bookTrip = document.querySelector('.book-trip');
+let calculateCost = document.querySelector('.calculate-cost');
 
 let tripsToDisplay, allDestinations, user;
 
 window.onload = fetchStuff;
 selectionBox.addEventListener('click', clickLog);
-bookTrip.addEventListener('click', validateForm);
+calculateCost.addEventListener('click', validateForm);
 
 function fetchStuff() {
```

Comment:
```suggestion
function fetchStuff(userId) {
```

Reply:
Maybe for now we have `fetchStuff` have a userId parameter, and we hardcode it as an argument when we call `fetchStuff(7)` instead of inside our function.

Reply:
Yeah baby! I think this was the plan all along, all along.